### DISCUSSION TITLE
water flood sensor

### DISCUSSION THREAD

**Discussion by Mareckil** [jomjol/AI-on-the-edge-device#2965] [Ideas]
Well first of all thanks for a great project, It works well for me so far. But I do have an idea. As this project is strictly connected with water meeters and they are mostly in cellars or similar places where existing risk of flood. It would be great if we can use some GPIO to connect flood sensor and report it to MQTT. So we do not have to install separate devices.

**Comment by Slider0007** [comment]
This is already supported. Activate GPIO functionality and enable pin state forwarding to MQTT
